build.rs

```diff
@@ -286,6 +286,20 @@ fn test_parse_nasm_version() {
   assert_eq!((2, 14, 0), parse_nasm_version(ver_str));
 }
 
+#[cfg(feature = "asm")]
+fn is_nasm_installed_check() {
+  let mut cmd = std::process::Command::new("nasm");
+  cmd.arg("-h");
+  match cmd.output() {
```

Comment:
This is certainly minor in the overall scheme of things, but you can use `cmd.status()` instead which will be more performant as it avoids capturing stdout and stderr.

Reply:
Corrected